generators/rest-client-core/src/test/java/uk/gov/justice/services/clients/core/RestClientProcessorBuilder.java

```diff
@@ -25,7 +30,10 @@ public RestClientProcessor build() {
         final RestClientProcessor restClientProcessor = new RestClientProcessor();
         restClientProcessor.stringToJsonObjectConverter = new StringToJsonObjectConverter();
         restClientProcessor.jsonObjectEnvelopeConverter = new JsonObjectEnvelopeConverter();
-        restClientProcessor.enveloper = new Enveloper(null);
+        final ObjectMapper objectMapper = new ObjectMapperProducer().objectMapper();
+        final ObjectToJsonValueConverter converter = new ObjectToJsonValueConverter(objectMapper);
+        final Enveloper enveloper =  new Enveloper(new UtcClock(), converter);
```

Comment:
Isn't this what the EnveloperFactory is for? Creating real envelopers for tests?